addons/mail/static/src/models/channel_member_list_view.js

```diff
@@ -2,11 +2,31 @@
 
 import { registerModel } from '@mail/model/model_core';
 import { one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ChannelMemberListView',
     identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeChannel() {
+            if (this.chatWindowOwner) {
+                return replace(this.chatWindowOwner.thread);
+            }
+            if (this.threadViewOwner) {
+                return replace(this.threadViewOwner.thread);
+            }
+            return clear();
+        },
+    },
     fields: {
+        channel: one('Thread', {
+            compute: '_computeChannel',
+            required: true,
```

Comment:
Required on compute seams strange.But maybe you want `readonly` ?

Reply:
Changes done.